gunrock/graph/graph_base.cuh

```diff
@@ -37,6 +37,7 @@ enum : GraphFlag {
   HAS_CSC = 0x0200,
   HAS_COO = 0x0400,
   HAS_GP = 0x0800,
+  HAS_DYN = 0x1000,
 
   GRAPH_PINNED = 0x1000,
```

Comment:
Is this going to be an issue (same value for the enum)?

Reply:
Nice catch. Fixed https://github.com/gunrock/gunrock/pull/713/commits/f937024a6415054821e33efb6d292d71ea4d323c.